ci-operator/step-registry/ipi/install/install/aws/ipi-install-install-aws-commands.sh

```diff
@@ -41,6 +41,22 @@ function prepare_next_steps() {
       "${dir}/metadata.json"
 }
 
+function wait_router_lb_provision() {
+    local try=0 retries=20
+
+    while [ X"$(oc -n openshift-ingress get service router-default -o json | jq -r '.status.loadBalancer.ingress[][]')" == X"" ] && [ $try -lt $retries ]; do
+        echo "router lb is not gernerated, waiting..."
+        sleep 30
+        try=$(expr $try + 1)
+    done
+    if [ X"$try" == X"$retries" ]; then
+        echo "ERROR: router lb is not ready, pls check your cluster - 'oc get svc -n openshift-ingress'"
```

Comment:
I expect most folks will recognize "pls", but it would probably be easier to read as "please".  But... we aren't actually asking a human to do anything here, are we?  We're about to run the Service dump (might want to dump `-o yaml`?) in the step logic.  So maybe something like:

```sh
SERVICE="$(oc -n openshift-ingress get service router-default -o json)"
while test -z "$(echo "${SERVICE}" | jq -r '.status.loadBalancer.ingress[][]')" ] && [ "${try}" -lt "${retries}" ]; do
  echo "waiting on router-default service load balancer ingress..."
  sleep 30
  SERVICE="$(oc -n openshift-ingress get service router-default -o json)"
  try="((try + 1))"
done
if [ "$try" -eq "$retries" ]; then
  echo "${SERVICE}"
  echo "ERROR: router-default service failed to provision load balancer ingress"
  return 4
fi
```

or similar?